src/pages/index.module.scss

```diff
@@ -5,17 +5,76 @@
 }
 
 body {
-  font-family: "Open Sans", sans-serif;
-}
-
-.titleHeader {
-  text-align: center;
-  padding-top: 25%;
-  h1 {
-    font-size: 42px;
-    margin-bottom: 24px;
-  }
-  a {
-    display: inline-block;
-  }
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 400;
+}
+
+h1 {
+  text-align: left;
+  position: absolute;
+  width: 408px;
+  height: 95px;
+  left: 228px;
```

Comment:
**nitpick:** You don't need such params as `left, height, top` in your header. I know it's in FIgma, but these are mostly for Figma purposes, so it can be displayed correctly there. Let's remove it from here as it adds unnecessary noise